tests/locator_test.go

```diff
@@ -19,18 +19,18 @@ func TestLocatorClick(t *testing.T) {
 	// Selecting a single element and clicking on it is OK.
 	t.Run("ok", func(t *testing.T) {
 		result := func() bool {
-			cr := p.Evaluate(tb.toGojaValue(`() => window.result`))
-			return cr.(goja.Value).ToBoolean() //nolint:forcetypeassert
+			ok := p.Evaluate(tb.toGojaValue(`() => window.result`))
+			return ok.(goja.Value).ToBoolean() //nolint:forcetypeassert
 		}
-		link := p.Locator("#link", nil)
-		link.Click(nil)
+		l := p.Locator("#link", nil)
+		l.Click(nil)
```

Comment:
Avoid the variable altogether?

```suggestion
		p.Locator("#link", nil).Click(nil)
```

Same in the other places where it makes sense (`Dblclick`, etc.).

Reply:
Please see: 8e31fec